keras_nlp/models/bart/bart_presets.py

```diff
@@ -71,3 +71,40 @@
         "merges_hash": "75a37753dd7a28a2c5df80c28bf06e4e",
     },
 }
+
+
+seq_2_seq_lm_presets = {
+    "bart_large_en_cnn": {
```

Comment:
Hmm, this is our first time facing this question, but I actually think we can just ship this as a backbone right? Because there are no new weights?

I don't see any disadvantage, and it would have the advantage of being able to load this on a backbone and run absolutely any workflow on top (e.g. your own "task flow" with custom code if you so chose).